src/target_info.rs

```diff
@@ -5,10 +5,21 @@ pub const RISCV_ARCH_MAPPING: &[(&str, &str)] = &[
     ("riscv32gc", "riscv32"),
     ("riscv32i", "riscv32"),
     ("riscv32im", "riscv32"),
-    ("riscv32ima", "riscv32"),
     ("riscv32imac", "riscv32"),
     ("riscv32imafc", "riscv32"),
     ("riscv32imc", "riscv32"),
     ("riscv64gc", "riscv64"),
     ("riscv64imac", "riscv64"),
 ];
+pub const WINDOWS_TRIPLE_MAPPING: &[(&str, &str)] = &[
+    ("aarch64-pc-windows-gnullvm", "aarch64-pc-windows-gnu"),
+    ("aarch64-uwp-windows-msvc", "aarch64-pc-windows-msvc"),
+    ("i686-pc-windows-gnullvm", "i686-pc-windows-gnu"),
+    ("i686-uwp-windows-gnu", "i686-pc-windows-gnu"),
+    ("i686-uwp-windows-msvc", "i686-pc-windows-msvc"),
+    ("i686-win7-windows-msvc", "i686-pc-windows-msvc"),
```

Comment:
@roblabla i686 and x86_64 have different vendor in their `llvm_triple`, is it intended?
https://github.com/rust-lang/rust/pull/118150/files#diff-bdd6b94a9eff21c9a3c3d1bb987409f8c4f2fc86c3ba7fccaa24759dd46975e9R11

Reply:
Not sure cc @ChrisDenton can you take a look at this please?

Reply:
That looks like a rustc bug to me. LLVM does not abuse the vendor field like rustc does so it should always be `pc`.

Reply:
Ah yeah that's on me, it should definitely say pc in there.

Reply:
I just opened PR https://github.com/rust-lang/rust/pull/133239 for this.